source/common/access_log/access_log_formatter.cc

```diff
@@ -779,17 +788,34 @@ PlainStringFormatter::PlainStringFormatter(const std::string& str) { str_.set_st
 std::string PlainStringFormatter::format(const Http::RequestHeaderMap&,
                                          const Http::ResponseHeaderMap&,
                                          const Http::ResponseTrailerMap&,
-                                         const StreamInfo::StreamInfo&) const {
+                                         const StreamInfo::StreamInfo&, absl::string_view) const {
   return str_.string_value();
 }
 
 ProtobufWkt::Value PlainStringFormatter::formatValue(const Http::RequestHeaderMap&,
                                                      const Http::ResponseHeaderMap&,
                                                      const Http::ResponseTrailerMap&,
-                                                     const StreamInfo::StreamInfo&) const {
+                                                     const StreamInfo::StreamInfo&,
+                                                     absl::string_view) const {
   return str_;
 }
 
+std::string LocalReplyBodyFormatter::format(const Http::RequestHeaderMap&,
+                                            const Http::ResponseHeaderMap&,
+                                            const Http::ResponseTrailerMap&,
+                                            const StreamInfo::StreamInfo&,
+                                            absl::string_view local_reply_body) const {
+  return local_reply_body.data();
```

Comment:
Is this right? I don't think string_view is guaranteed to be null terminated.

Reply:
Done